seml/observers.py

```diff
@@ -1,10 +1,17 @@
 import urllib.parse
 import logging
+from sacred.observers.base import RunObserver, td_format
+from sacred.config.config_files import load_config_file
+from bson import json_util
+import json
+from datetime import datetime, timedelta, timezone
+import re
 
 from seml.database import get_mongodb_config
 from seml.settings import SETTINGS
 
-__all__ = ['create_mongodb_observer', 'create_slack_observer', 'create_neptune_observer']
+__all__ = ['create_mongodb_observer', 'create_slack_observer', 'create_neptune_observer', 'create_mattermost_observer',
+           'MattermostObserver']
```

Comment:
Should we expose just `create_mattermost_observer` here and not the full `MattermostObserver`? Would seem more consistent to me.